chart/templates/worker/_container.tpl

```diff
@@ -7,6 +7,7 @@
   imagePullPolicy: {{ .Values.images.pullPolicy }}
   env:
   {{ include "envAssets" . | nindent 2 }}
+  {{ include "envS3" . | nindent 2 }}
```

Comment:
no other containers need these environment variables? the jobs, the api/admin services, the storage machine?

Reply:
S3Client is initialized only in `worker` `rows` and `search`. I think we don't need it in other containers.

Reply:
OK, thanks for checking